notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,581 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bac00b07-32db-45c3-80f6-8243ae2c5446",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "173ccbd7-a08b-4d47-b1ff-f8b0f9886295",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q \"pyclipper>=1.2.1\" \"shapely>=1.7.1\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cd695666-f8b8-4823-9c8f-70b7d5b14be2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "import sys\n",
+    "import math\n",
+    "from pathlib import Path\n",
+    "import tarfile\n",
+    "import requests\n",
+    "import copy\n",
+    "\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "import processing as processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9386c70f-c383-48d3-a6b4-31b4ceb1653b",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## PaddleOCR with OpenVINO™\n",
+    "\n",
+    "[PaddleOCR](https://github.com/PaddlePaddle/PaddleOCR) is an ultra-light OCR model trained with PaddlePaddle deep learning framework, which aims to create multilingual and practical OCR tools. \n",
+    "\n",
+    "The PaddleOCR pre-trained model used in the demo refers to the *\"Chinese and English ultra-lightweight PP-OCR model (9.4M)\"*. More open source pre-trained models can be downloaded at [PaddleOCR Github](https://github.com/PaddlePaddle/PaddleOCR) or [PaddleOCR Gitee](https://gitee.com/paddlepaddle/PaddleOCR).\n",
+    "\n",
+    "A standard PaddleOCR includes two parts of deep learning models, text detection and text recognition. This notebook only needs the text recognition part. For running the model, we first initialize the runtime for inference, then, read the network architecture and model weights from the `.pdmodel` and `.pdiparams` files to load to CPU.\n",
+    "\n",
+    "More details for running PaddleOCR with OpenVINO™ are shown in [405-paddle-ocr-webcam](../405-paddle-ocr-webcam/405-paddle-ocr-webcam.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "985177b2-2eac-4dc4-8c62-68e57352a172",
+   "metadata": {},
+   "source": [
+    "### Download the Model for Text **Recognition**\n",
+    "\n",
+    "The pre-trained models used in the demo are downloaded and stored in the \"model\" folder."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ec981e3-6d52-4de9-82dc-10506574abfd",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Define the function to download text detection and recognition models from PaddleOCR resources.\n",
+    "\n",
+    "def run_model_download(model_url, model_file_path):\n",
```

Comment:
I have already a built a reusable function for file downloading : ```from notebook_utils import download_file```. Can we switch to it ?